flash.py

```diff
@@ -1,4 +1,10 @@
 import os
-
+import subprocess
+PROGRAMMER = "sammo"
```

Comment:
Little issue here. This is referring to the micro-controller programmer, as in the way we put code onto the chip. It should be `"usbasp"`. If you'd like to credit yourself please put your name in a comment in the top of the script.